README.md

```diff
@@ -16,14 +16,20 @@ Add the following to your `Cargo.toml`:
 stylist = "0.10"
 ```
 
+By default, the `yew_integration` feature is enabled. If you want to use stylist outside of `yew`,
```

Comment:
While this default is not in a release maybe this should be consistent with the version below in the README. Maybe refer to a `git` version in the docs on `master`?